impeller/tessellator/tessellator.cc

```diff
@@ -468,6 +468,34 @@ EllipticalVertexGenerator Tessellator::FilledEllipse(
                                    });
 }
 
+EllipticalVertexGenerator Tessellator::FilledRoundRect(
+    const Matrix& view_transform,
+    const Rect& bounds,
+    const Size& radii) {
+  if (radii.width * 2 < bounds.GetSize().width &&
```

Comment:
This should be `||` not `&&`. Reverting.

Reply:
Revert wasn't clean so I fixed it instead: https://github.com/flutter/engine/pull/49021